app/src/main/java/org/mifos/mobile/ui/savings_account_withdraw/SavingsAccountWithdrawScreen.kt

```diff
@@ -0,0 +1,191 @@
+package org.mifos.mobile.ui.savings_account_withdraw
+
+import android.widget.Toast
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.mifos.mobile.R
+import org.mifos.mobile.core.ui.component.MifosOutlinedTextField
+import org.mifos.mobile.core.ui.component.MifosProgressIndicator
+import org.mifos.mobile.core.ui.component.MifosTopBar
+import org.mifos.mobile.core.ui.component.NoInternet
+import org.mifos.mobile.core.ui.theme.MifosMobileTheme
+import org.mifos.mobile.models.accounts.savings.SavingsWithAssociations
+import org.mifos.mobile.utils.Network
+import org.mifos.mobile.utils.getTodayFormatted
+
+@Composable
```

Comment:
Create another function above this 
@Composable
fun SavingsAccountWithdrawScreen(
    view: SavingsAccountWithdrawViewModel = hiltViewModel,
    navigateBack: (withdrawSuccess: Boolean) -> Unit,
    withdraw: (String) -> Unit,
    noInternet: () -> Unit,
) {

val uiState by viewModel.savingsAccountWithdrawUiState.collectAsStateWithLifecycle()
SavingsAccountWithdrawScreen(
    uiState,
    navigateBack,
    withdraw,
    noInternet,
)

}